src/pkg/packaging/dir.proj

```diff
@@ -55,6 +56,19 @@
     <Copy Sourcefiles="@(SharedFxFiles)"
           DestinationFolder="$(SharedFrameworkPublishRoot)shared/%(SharedFxFiles.RecursiveDir)" />
 
+    <!-- Copy nethost -->
+    <PropertyGroup>
+      <InstallerSourceOSPlatformConfig Condition="'$(InstallerSourceOSPlatformConfig)' == ''">$(OSPlatformConfig)</InstallerSourceOSPlatformConfig>
```

Comment:
Ah yeah, didn't consider this. I think it would be nice to move this into the highest shared dir.props, this group:

https://github.com/dotnet/core-setup/blob/c3c87f434540d184dd45d112846599305871cddd/src/pkg/dir.props#L76

combining with:

https://github.com/dotnet/core-setup/blob/c3c87f434540d184dd45d112846599305871cddd/src/pkg/projects/packaging-tools/packaging-tools.props#L4-L8

No big deal, would be nice to keep it in one place with the comment though.